code/modules/projectiles/guns/projectile/revolver.dm

```diff
@@ -45,7 +46,9 @@
 			CB.loc = get_turf(loc)
 			CB.SpinAnimation(10, 1)
 			CB.update_icon()
+			CB.play_casing_sound()
 			num_unloaded++
+			sleep(1)

```

Comment:
Why the 0.1 second sleep?

Reply:
Without it, all the cases drop at the same time, this could be fixed with the variable timer.